assets/javascripts/scripts.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * togglePressed() toggles the aria-pressed atribute between true or false
+ *
+ * @param ( id object) button to be operated on
+ *
+ * @return N/A
+ */
+function togglePressed(element) {
+
+  // reverse the aria-pressed state
+  if (element.attr('aria-pressed') == 'true') {
+    element.attr('aria-pressed', 'false');
+  }
+  else {
+    element.attr('aria-pressed', 'true');
+  }
+}
+
+var parseMediumJSON = function(post_rows) {
+  // max 10 posts
+  post_rows = post_rows || 10;
+
+  // get the template
+  var template_markup = $('#post-template').html();
+
+  // get the data
+  $.getJSON('https://8k431k2y7h.execute-api.us-west-2.amazonaws.com/Production/medium', function(data) {
```

Comment:
If we're using this in two places we should think of a good strategy to keep them in sync.